website/docs/developer/research/research-reconciliation.mdx

```diff
@@ -692,48 +678,71 @@ items for managing jobs.
+
+This table would capture the discrepancies for reporting out to the operators
+about files that are missing from ORCA S3 or have different metadata values than
+what is expected. Note that any entries with an orca_last_update after the job's
+start_time may show as an error due to race conditions with the catalog update
+process, and should be displayed as such when reporting results.
 
 | Attribute Name   | Data Type | Description                               | Notable Items   |
-| ---------------- | --------- | ----------------------------------------- | --------------- |
-| job_id           | `int8`    | Reconcile job the report is a part of.    | FK constraint back to the jobs table with a many to one relationship. |
-| key_path         | `string`  | Key key value from S3 listing.            | |
-| etag             | `string`  | ETag key value from S3 listing.           | |
-| last_update      | `timestamptz` | LastModified key value from the S3 listing. | |
-| size_in_bytes    | `int8`    | Size key value from the S3 listing.       | |
-| storage_class    | `string`  | StorageClass key value from the S3 listing. | |
+| --------------------- | --------- | ----------------------------------------- | --------------- |
+| job_id                | `int8`    | Reconcile job the report is a part of.    | FK constraint back to the jobs table with a many to one relationship. |
+| orca_archive_location | `string`  | ORCA S3 Glacier bucket the file resides in. | From Catalog. |
+| collection_id         | `string`  | Collection ID from the ORCA Collection object the file belongs to. | From Catalog. |
+| granule_id            | `string`  | Granule ID from the ORCA Granule object the file belongs to. | From Catalog. |
+| file_name             | `string`  | File name from the ORCA File object the file belongs to. | From Catalog. |
+| key_path              | `string`  | Key path that includes the file name from the ORCA File object the file belongs to. | From Catalog. |
+| orca_etag             | `string`  | Etag value from the ORCA File object the file belongs to. | From Catalog. |
+| s3_etag               | `string`  | ETag key value from S3 listing.           | NULLABLE column |
+| orca_last_update      | `timestamptz` | Last Update value from the ORCA File object the file belongs to. | From Catalog. |
+| s3_last_update        | `timestamptz` | LastModified key value from the S3 listing. | NULLABLE column |
+| orca_size             | `int8`    | Size value from the ORCA File object the file belongs to. | From Catalog. |
+| s3_size               | `int8`    | Size key value from the S3 listing.       | NULLABLE column |
+| discrepancy_type      | `string`  | Type of discrepancy encountered like "missing" or "value x mismatch" | |
 
 
 ## Open Questions yet to be answered
 
 - What is the proper date to use when receiving a date range from Cumulus for a reconciliation report?
+  - Answer: Milliseconds since Unix Epoch.
```

Comment:
May want to put that the format is milliseconds since unix epoch. A date range for the reconciliation from Cumulus would use the `createdAt` date from Cumulus that is part of the granule record. The comparison would use the last update internally, I would think at the file level.

Reply:
I thought this was just about the format for some reason. Removed for now.